reference/5.1/Microsoft.PowerShell.Core/About/about_Jobs.md

```diff
@@ -15,34 +15,49 @@ expression in the background without interacting with the current session.
-layer adds overhead, including object serialization. All objects passed to and
-from the remote session must be serialized and then deserialized as it passes
-between the client and the target session. The serialization operation can use
-many compute and memory resources for large complex data objects.
-
-This topic explains how to run background jobs in PowerShell on a local
-computer. For information about running background jobs on remote computers,
-see [about_Remote_Jobs](about_Remote_Jobs.md). For more information about
-thread jobs, see
-[about_Thread_Jobs](/powershell/module/microsoft.powershell.core/about/about_Thread_Jobs).
-
-When you start a background job, the command prompt returns immediately, even
-if the job takes an extended time to complete. You can continue to work in the
-session without interruption while the job runs.
+PowerShell concurrently runs commands and scripts through jobs. There are three
+jobs types provided by PowerShell to support concurrency.
+
+- `RemoteJob` - Commands and scripts run on a remote session. For information,
+  see [about_Remote_Jobs](about_Remote_Jobs.md).
+- `BackgroundJob` - Commands and scripts run in a separate process on the local
+  machine.
+- `PSTaskJob` or `ThreadJob` - Commands and scripts run in a separate thread
+  within the same process on the local machine. For more information, see
+  [about_Thread_Jobs](/powershell/module/ThreadJob/about_Thread_Jobs).
+
+Running scripts remotely, on a separate machine or in a separate process,
+provides great isolation. Any errors that occur in the remote job do not affect
+other running jobs or the parent session that started the job. However, the
+remoting layer adds overhead, including object serialization. All objects are
+serialized and deserialized as they are passed between the parent session and
+the remote (job) session. Serialization of large complex data objects can
+consume large amounts of compute and memory resources and transfer large
+amounts of data across the network.
+
+Thread-based jobs are not as robust as remote and background jobs, because they
+run in the same process on different threads. If one job has a critical error
+that crashes the process, then all other jobs in the process are terminated.
+
+However, thread-based jobs require less overhead. They don't use the remoting
+layer or serialization. The result objects are returned as references to live
+objects in the current session. Without this overhead, thread-based jobs run
+faster and use fewer resources than the other job types.
+
+> [!IMPORTANT]
+> The parent session that created the job also monitors the job status and
+> collects pipeline data. The job child process is terminated by the parent
+> process once the job reaches a finished state. If the parent session is
+> terminated, all running child jobs are terminated along with their child
+> processes.
+
+There are two ways work around this limitation:
```

Comment:
I wouldn't call this a 'limitation'.  It is just a 'by design' behavior.  If a user wants different behavior, like 'fire and forget', then there is Invoke-Command session disconnect/reconnect, or Start-Process which really has nothing to do with jobs at all.

Reply:
fixed